SQL1/Homework1-SQL.sql

```diff
@@ -0,0 +1,173 @@
+CREATE DATABASE ZooManagement5;
+
+USE ZooManagement5;
+
+CREATE TABLE Animals1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfAnimal VARCHAR(30),
+    NumberOfAnimal INT,
+);
+
+CREATE TABLE Species1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfSpecies VARCHAR(20),
+    NumberOfSpecies INT,
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE CareTakers1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    FirstName VARCHAR(30),
+    LastName VARCHAR(30),
+    AreaOfWork VARCHAR(20),
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE ZooAreas1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfTheArea VARCHAR(20),
+    CareTakersId INT REFERENCES CareTakers1(Id),
+);
+
+CREATE TABLE Visitors1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    DayOfTheWeek VARCHAR(30),
+    NumbersOfVisitors INT,
+    AnimalsId INT REFERENCES Animals1(Id),
+);
+
+CREATE TABLE Tickets1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Validity VARCHAR(30),
+    Price MONEY,
+    VisitorsId INT REFERENCES Visitors1(Id),
+);
+
+CREATE TABLE ZooDetails1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    ZooName VARCHAR(30),
+    StreetName VARCHAR(30),

```

Comment:
The information about the Zoo address details should be stored in a different table maybe called Addresses.